package.json

```diff
@@ -9,14 +9,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "1x": "^0.0.0",
```

Comment:
why removing this dependency

Reply:
it was not removed but for some reason it was moved down a couple of lines to to `line 19`